pandas/tests/extension/test_arrow.py

```diff
@@ -1835,7 +1835,7 @@ def test_str_fullmatch(pat, case, na, exp):
 
 @pytest.mark.parametrize(
     "sub, start, end, exp, exp_typ",
-    [["ab", 0, None, [0, None], pa.int32()], ["bc", 1, 3, [2, None], pa.int64()]],
+    [["ab", 0, None, [0, None], pa.int32()], ["bc", 1, 3, [1, None], pa.int64()]],
```

Comment:
```
>>> s = pd.Series(["abc"], dtype=pd.StringDtype())
>>> s.str.find("bc", 1, 3)
0    1
dtype: Int64
>>>
```

This would now match the result produced by pd.StringDtype()